unfurl/parsers/parse_youtube.py

```diff
@@ -22,11 +25,32 @@
 }
 
 
+def format_seconds(seconds):
+    if seconds < 60:
+        return f"{seconds} seconds"
+    elif seconds < 3600:
+        return time.strftime("%M:%S", time.gmtime(seconds)) + " minutes"
+    else:
+        return time.strftime("%H:%M:%S", time.gmtime(seconds)) + " hours"
+
+
 def run(unfurl, node):
     youtube_domains = ['youtube.com', 'youtu.be']
     if any(youtube_domain in unfurl.find_preceding_domain(node) for youtube_domain in youtube_domains):
-        if node.key == 't':
-            param_text = f'The youtube video will begin playing at {str(node.value)} seconds.'
+        if node.key == 't' or node.key == 'time_continue' or node.key == 'start':
+            try:
```

Comment:
Added try except because parameter "t" doesn't have to be in seconds, which will cause an exception.

Example: https://www.youtube.com/watch?v=5VKTaFBlMcE&feature=youtu.be&t=07m03s

Reply:
Ah interesting, I wasn't aware it could appear like that